### PR TITLE
💬 Add admin email link to error messages.

### PR DESCRIPTION
**Pull request by anncatton**
Adds an email link to api token and configuration errors. Related #63

### CODE REVIEW

File: components/pages/user/ApiTokenInfo.tsx [hunk 1]
```diff
@@ -288,7 +291,14 @@ const ApiTokenInfo = () => {
...
             >
-              {errorMessage.message.toString()}
+              {typeof errorMessage === 'string' ? (
+                <span>
+                  {errorMessage}
+                  <GenericHelpMessage />
+                </span>
+              ) : (
+                errorMessage
+              )}
```
Comment (ciaranschutte) [inline components/pages/user/ApiTokenInfo.tsx:301]: I don't quite understand this.
Why is displaying a help message based on whether another error is a string or not?
@anncatton

Comment (anncatton) [reply components/pages/user/ApiTokenInfo.tsx:301]: the string errorMessages need that `GenericHelpMessage` appended to them. the one errorMessage that is created as a ReactNode is worded differently so i couldn't just append like the others.

File: components/pages/explorer/getConfigError.tsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import StyledLink from '../../Link';
 
 import { getConfig } from '../../../global/config';
 import { Project } from '.';
+import { GenericHelpMessage } from '../../DMSAdminContact';
```
Comment (joneubank) [inline components/pages/explorer/getConfigError.tsx:10]: Do we also have `WeirdlySpecificHelpMessage`?

Comment (anncatton) [reply components/pages/explorer/getConfigError.tsx:10]: Anything is possible

**Review by ciaranschutte — APPROVED** [review]
Awesome! great coverage of error states

**Pull request merged**